# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../util-linux/disable_cramfs_tools.diff
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- disk-utils/Makefile.orig	Wed Jan 15 20:15:21 2003
+++ disk-utils/Makefile	Wed Jan 15 20:15:49 2003
@@ -20,7 +20,7 @@
 
 ETC=	fdprm
 
-MAYBE=  setfdprm raw fsck.cramfs mkfs.cramfs
+MAYBE=  setfdprm raw
 
 ifneq "$(HAVE_FDUTILS)" "yes"
 USRBIN:=$(USRBIN) setfdprm
@@ -32,21 +32,7 @@
 MAN8:=$(MAN8) raw.8
 endif
 
-ifeq "$(HAVE_ZLIB)" "yes"
-SBIN:=$(SBIN) fsck.cramfs mkfs.cramfs
-endif
-
 all: $(SBIN) $(USRBIN)
-
-fsck.cramfs: fsck.cramfs.o
-	$(CC) $(LDFLAGS) -o fsck.cramfs fsck.cramfs.o -lz
-
-mkfs.cramfs: mkfs.cramfs.o $(LIB)/md5.o
-	$(CC) $(LDFLAGS) -o mkfs.cramfs mkfs.cramfs.o $(LIB)/md5.o -lz
-
-mkfs.cramfs.o: $(LIB)/md5.h
-
-fsck.cramfs.o mkfs.cramfs.o: cramfs.h
 
 fsck.minix.o mkfs.minix.o: bitops.h minix.h
 
